Лабораторная работа 1/task1.py

```diff
@@ -0,0 +1,8 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+
+# TODO заменить значение пропущенного элемента средним арифметическим
+left = 4
+right = 5
+arithmetic_mean = sum(numbers[:left] + numbers[right:]) / (len(numbers[:left] + numbers[right:]) +1)
```

Comment:
Тогда в этой строке мы будем опираться на найденный индекс искомого значения, а также вспомним условие задачи - находить среднее арифметическое необходимо от всего количестве элементов списка. Да, `None` это не число, поэтому в поиске суммы значений мы его исключаем, а вот при подсчете общего числа элементов списка, исключать его нельзя. 
```suggestion
arithmetic_mean = (sum(numbers[:none_index] + numbers[none_index + 1:])) / len(numbers) 
```